### ISSUE TITLE
Consolidate efforts

### ISSUE THREAD

**Issue by balupton** [bevry/staticsitegenerators-list#13]
- http://staticsitegenerators.net/ uses this listing
- http://staticgen.com/ https://github.com/netlify/staticgen
- https://github.com/pinceladasdaweb/Static-Site-Generators
- http://www.mzlinux.org/?q=node/415
- http://modernstatic.com/
- http://bundlr.com/b/static-website-generators
- https://gist.github.com/max-arnold/6402386

**Comment by balupton** [comment]
Maybe as a fun hack day project a script could be written to generate a markdown file from our compiled JSON listing to generate a markdown file in the https://github.com/pinceladasdaweb/Static-Site-Generators format, as a double smackdown for the uselessness of that listing

**Comment by jonschlinkert** [comment]
>  compiled JSON listing

From [this YAML file](https://github.com/bevry/staticsitegenerators-list/blob/master/list.yaml)?

**Comment by balupton** [comment]
Correct. The website enhances it into https://staticsitegenerators.net/list.json

**Comment by SurajShukla** [comment]
Please let me know if I couple be of any help.

**Comment by balupton** [comment]
@SurajShukla will likely reach out to you as soon as a decision has been made on #220

**Comment by davisonio** [comment]
Some great resources here which we could consolidate:
- https://github.com/aharris88/awesome-static-website-services

**Comment by balupton** [comment]
@davisonio I've had that desire before but never managed to figure out how, just scanning for and comparing URLs is a fantastic and easy implementable idea, good thinking!
